wrappers/kn/src/nativeMain/kotlin/zxingcpp/ImageView.kt

```diff
@@ -5,42 +5,63 @@
 
 package zxingcpp
 
+import cnames.structs.ZXing_Image
 import cnames.structs.ZXing_ImageView
 import kotlinx.cinterop.*
 import zxingcpp.cinterop.*
 import kotlin.experimental.ExperimentalNativeApi
+import kotlin.native.ref.Cleaner
 import kotlin.native.ref.createCleaner
 
-@OptIn(ExperimentalForeignApi::class)
+@OptIn(ExperimentalForeignApi::class, ExperimentalNativeApi::class)
 class ImageView(
-	val data: ByteArray,
```

Comment:
I would prefer we left this class as it was before. The `asImageView()` function is only useful for the testing use case. And the test based on that does not really add too much value, since it tests the functionality of the underlying c++ code but that is tested already elsewhere. We need to test if the member accessors of `Image` produce something correct. For that it is more helpful to get that data out and construct a new `ImageView` and then pass that back in.

Reply:
done